core-bundle/tests/Twig/Interop/ContaoEscaperTest.php

```diff
@@ -71,6 +73,8 @@ public function provideHtmlInput(): \Generator
     }
 
     /**
+     * @param string|int $input
+     *
      * @dataProvider provideHtmlAttributeInput
      */
     public function testEscapesHtmlAttributes($input, string $expectedOutput): void
```

Comment:
Many of these can be changed I think.

```suggestion
     * @dataProvider provideHtmlAttributeInput
     */
    public function testEscapesHtmlAttributes(string $input, string $expectedOutput): void
```

Reply:
Changed in 4856530db904f2246a3fe79850e1cdf76217efa9. However, the others in the class cannot be changed, because `provideHtmlInput()` returns `123` in line 60.